test/CMakeLists.txt

```diff
@@ -2,290 +2,74 @@
 # Tests configuration
 #
 
-set(UVW_SRC_DIR ${CMAKE_SOURCE_DIR}/${PROJECT_SRC_DIR})
+set(UVW_SRC_DIR ${PROJECT_SRC_DIR})
+
+if(WIN32)
+    set(WINSOCK2 ws2_32)
+elseif(NOT APPLE)
```

Comment:
Shouldn't you look for `rt` only on Windows? This way it searches it also on linux.